app/views/RoomView/index.js

```diff
@@ -531,6 +531,9 @@ class RoomView extends React.Component {
 	}
 
 	onEditInit = (message) => {
+		if (message.attachments.length > 0) {
+			message._raw.msg = message.attachments[0].description;
```

Comment:
`message` on this context is an observable from WatermelonDB, so it shouldn't be changed directly (it's error prone).
Instead, you can pick only the properties needed from the object to do the edit.
Also, are you sure this is going to work?

Reply:
I tested it and it is work well, but like you said I need change the code to make sure the db is not change directly